gapic-generator/templates/default/lib/_package.erb

```diff
@@ -6,13 +6,24 @@ require "<%= service.service_require %>"
 require "<%= package.gem.version_require %>"
 <% end %>
 <%- unless package.empty? -%>
+<%- if package.services.first.generate_grpc_clients? -%>
+<%- grpc_client_description = package.services.first.generate_rest_clients? ? "GRPC client" : "client" -%>
```

Comment:
Logic like this can be moved to a presenter.

Reply:
:+1: